### ISSUE TITLE
No attachment

### ISSUE THREAD

**Issue by peterjaap** [judges119/readlater#1]
Hi,

I set up my repo with CI/CD and it looks like it works;

![image](https://user-images.githubusercontent.com/431360/132492127-ba52693e-7ab0-4e0d-8c27-05e9f416781b.png)

However, I get this mail in my "From" email address inbox from Amazon:

> Dear Customer,
> 
> The message you sent to xxxxxxxxxx@kindle.com 10:17 AM on Wednesday, September 08, 2021 GMT did not contain any attached documents or image files.

Looks like the attachment isn't added. I do see it in my artifacts though;

![image](https://user-images.githubusercontent.com/431360/132492278-c55e8fd5-ebfa-43de-8bd0-8fde171e28e1.png)

Any idea?

P.s. the .gitlab-ci.yml on your blog is outdated when compared to the one in this repo (I first copied the code from the blog, then found errors and copied the one from here).

**Comment by judges119** [comment]
Thanks for the heads up about the blog, will get that updated soon.

Are you getting a readlater.msg artifact in the `sendEmailToKindle` job in the pipeline? What does that look like, is it completely missing the base64 section or is the multipart boundary mangled at all?

**Comment by peterjaap** [comment]
@judges119 yes I'm getting that and there's a base64 string in there.

**Comment by judges119** [comment]
@peterjaap if you decode the base 64 string can you pull out the .mobi file?

if so, it looks like an attachment is present and i'm wondering if your server is stripping it? you could replace your kindle email with an email you control and see if it arrives with attachment (you might need to download the raw email message to confirm).

**Comment by peterjaap** [comment]
@judges119 yeah when I send it to my email address, and then forward it, it gets accepted. So which server strips it and why...
